topo/node/node.go

```diff
@@ -72,6 +75,11 @@ func Register(t topopb.Node_Type, fn NewNodeFn) {
 	mu.Unlock()
 }
 
+type NodeStatus struct {
```

Comment:
type Status since it will read as node.Status in other contexts

Reply:
Will update.